rest/usage-records/list-get-example-4/list-get-example-4.6.x.py

```diff
@@ -13,3 +13,8 @@
 records = client.usage.records.daily.list(
     category="calls-inbound", start_date="2012-09-01", end_date="2012-09-30"
 )
+
+# Print the JSON object
```

Comment:
I missed this in your previous PR, but I'd suggest omitting this comment here and in the other PRs (the `record` objects aren't JSON objects -- they've been deserialized into Record objects from the response JSON). It's clear what's happening and most of our other code samples also don't have a comment before the print lines.